seacrowd/sea_datasets/bhinneka_korpus/bhinneka_korpus.py

```diff
@@ -0,0 +1,181 @@
+Indonesia local language and approximately 3,000 sentences for other languages, and one lexicon dataset creation for 
+Beaye language. In addition, since beaye is a undocumented language, we don't have any information yet about the use 
+of language code. Therefore, we used "day" (a code for land dayak language family) to represent the language."""
+
+_HOMEPAGE = "https://github.com/joanitolopo/bhinneka-korpus"
+_LICENSE = Licenses.APACHE_2_0.value
+_URLS = "https://raw.githubusercontent.com/joanitolopo/bhinneka-korpus/main/"
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+_LANGUAGES = ["abs", "day", "mkn", "aoz", "mak"]
+LANGUAGES_MAP = {
+    "abs": "ambonese-malay",
+    "day": "beaye",
+    "mkn": "kupang-malay",
+    "aoz": "uab-meto",
+    "mak": "makassarese"
+}
+
+
+def process_list(data):
+    if isinstance(data, list):
+        if len(data) == 1 and isinstance(data[0], str):
+            return data[0]  # Return the single string if it's a list containing only one string
+        else:
+            return data  # Return the list as is if it contains multiple strings or other elements
+    else:
+        return data
+
+
+class BhinnekaKorpus(datasets.GeneratorBasedBuilder):
+    """A Collection of Multilingual Parallel Datasets for 5 Indonesian Local Languages."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+    SEACROWD_SCHEMA_NAME = "t2t"
+
+    dataset_names = sorted([f"bhinneka_korpus_{lang}" for lang in _LANGUAGES])
+    BUILDER_CONFIGS = []
+    for name in dataset_names:
+        source_config = SEACrowdConfig(
+            name=f"{name}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=name
+        )
+        BUILDER_CONFIGS.append(source_config)
+        seacrowd_config = SEACrowdConfig(
+            name=f"{name}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=name
+        )
+        BUILDER_CONFIGS.append(seacrowd_config)
+
+    BUILDER_CONFIGS.extend(
+        [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_source",
+                version=SOURCE_VERSION,
+                description=f"{_DATASETNAME} source schema (all)",
+                schema="source",
+                subset_id=_DATASETNAME
+            ),
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+                version=SEACROWD_VERSION,
+                description=f"{_DATASETNAME} SEACrowd schema (all)",
+                schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+                subset_id=_DATASETNAME
+            )
+        ]
+    )
```

Comment:
I don't think we need all and per-language datasets, just implement either just per-language or just all.